tlsproxy_test.go

```diff
@@ -70,8 +70,8 @@ func TestProxy(t *testing.T) {
 	}
 	defer cl.Close()
 
-	go runServer(sl, l.Addr().String(), 150*time.Millisecond, serverConfig)
-	go runClient(cl, sl.Addr().String(), 150*time.Millisecond, clientConfig)
+	go RunServer(sl, l.Addr().String(), 150*time.Millisecond, serverConfig)
```

Comment:
I fixed this so that the unit test compiles and runs.

Reply:
Yeah I fixed that too, thanks